lib/graphql/enum_type.rb

```diff
@@ -136,7 +138,7 @@ def initialize(name:, description:, deprecation_reason:, value:)
         @name = name
         @description = description
         @deprecation_reason = deprecation_reason
-        @value = value
+        @value = value || name
```

Comment:
Ohhh I see, yeah, I agree with your judgment to fix this in `loader.rb` -- we shouldn't look for a hash key that doesn't exist (and never will)!